internal/k8s/epslices/endpoint_slices.go

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package epslices
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	discovery "k8s.io/api/discovery/v1beta1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type EpsOrSlices struct {
+	Type      EpsOrSliceType
+	EpVal     *v1.Endpoints
+	SlicesVal []discovery.EndpointSlice
+}
+
+type EpsOrSliceType int
+
+const (
+	Unknown EpsOrSliceType = iota
+	Eps
+	Slices
+)
+
+const SlicesServiceIndexName = "ServiceName"
```

Comment:
I think it's not used

Reply:
we use it here https://github.com/fedepaol/metallb/blob/0f608cd2638ae4e45bd50cc580a1d236dfeb50f1/internal/k8s/k8s.go#L179 and here https://github.com/fedepaol/metallb/blob/fcc7c64bdb47ab702cfb94af6c924a3312302ad1/internal/k8s/controllers/endpoints.go#L21

Those are coming in a sequent commit, I split the commits afterwards so that could be the reason.